From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
Date: Thu, 10 Nov 2022 21:49:36 +0100
Subject: [ARCHEOLOGY] Moving patches to per board and removing obsolete
 (#4409)

> X-Git-Archeology: - Revision f6a09def35e647a5442e0f92f399485be29f19a9: https://github.com/armbian/build/commit/f6a09def35e647a5442e0f92f399485be29f19a9
> X-Git-Archeology:   Date: Thu, 10 Nov 2022 21:49:36 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Moving patches to per board and removing obsolete (#4409)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 58d80414057b57359d60fa91293b82ac734efc06: https://github.com/armbian/build/commit/58d80414057b57359d60fa91293b82ac734efc06
> X-Git-Archeology:   Date: Tue, 13 Jun 2023 17:03:41 +0200
> X-Git-Archeology:   From: Ricardo Pardini <ricardo@pardini.net>
> X-Git-Archeology:   Subject: `uboot`: introduce `u-boot-sunxi-v2022.04` for Allwinner boards using 22.04 u-boot (`nanopineo`)
> X-Git-Archeology:
---
 configs/nanopi_neo_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/nanopi_neo_defconfig b/configs/nanopi_neo_defconfig
index 776972f7bf..9c49358405 100644
--- a/configs/nanopi_neo_defconfig
+++ b/configs/nanopi_neo_defconfig
@@ -7,6 +7,7 @@ CONFIG_DRAM_CLK=408
 # CONFIG_VIDEO_DE2 is not set
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_CONSOLE_MUX=y
+CONFIG_SYS_CLK_FREQ=480000000
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
-- 
Armbian

